src/main/java/org/alliancegenome/curation_api/jobs/executors/GeneMolecularInteractionExecutor.java

```diff
@@ -37,11 +37,11 @@ public void execLoad(BulkLoadFileHistory bulkLoadFileHistory) {
 
 			List<Long> interactionIdsLoaded = new ArrayList<>();
 			List<Long> interactionIdsBefore = geneMolecularInteractionDAO.findAllIds().getResults();
-			
+
 			bulkLoadFileHistory.setCount(interactionData.size());
 			updateHistory(bulkLoadFileHistory);
-			
-			boolean success = runLoad(geneMolecularInteractionService, bulkLoadFileHistory, null, interactionData, interactionIdsLoaded, false);
+
+			boolean success = runLoad(geneMolecularInteractionService, bulkLoadFileHistory, null, interactionData, interactionIdsLoaded, false, "Records");
```

Comment:
Shouldn't need to pass in "Records" here

Reply:
Done